dev/airflow.Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM apache/airflow:latest
+
+RUN airflow db migrate
+
+RUN airflow users create \
+    --username rdwatch \
+    --password rdwatch \
```

Comment:
this is hardcoded so in the .env file should we make a note of this?  It isn't like other containers where the username/password is parameterized.  Or maybe we should use the airflow environment variables?

https://airflow.apache.org/docs/apache-airflow/stable/howto/docker-compose/index.html#environment-variables-supported-by-docker-compose

Reply:
I saw this, but I couldn't get it to work so I just gave up and did that command in the docker file. Does it work for you?

Reply:
took a little searching but I think you can remove the local airflow DockerFile.  These `_` based commands are only meant for local development, but I assume your infrastructure code handles all this.
 I wouldn't worry about implementation now, just logging this comment for the future.
```
  airflow:
    image: apache/airflow:slim-latest-python3.12
    volumes:
      - airflow-data:/opt/airflow
      - type: bind
        source: ${PWD}/dev/airflow_sample_dag.py
        target: /opt/airflow/dags/airflow_sample_dag.py
    environment:
      AIRFLOW_HOME: "/opt/airflow/"
      _AIRFLOW_DB_MIGRATE: True
      _AIRFLOW_WWW_USER_CREATE: True
      _AIRFLOW_WWW_USER_USERNAME: "rdwatch"
      _AIRFLOW_WWW_USER_PASSWORD: "rdwatch"
    command: ["airflow", "standalone"]
    ports:
      - 8002:8080
    healthcheck:
      test: ["CMD", "curl", "--fail", "http://localhost:8080/health"]
      interval: 30s
      timeout: 10s
      retries: 5
      start_period: 30s
```